glom/test/test_basic.py

```diff
@@ -231,9 +231,9 @@ def test(*a, **kw):
         'kwargs': {'a': 'a'},
         'c': 'C',
     }
-    spec = Invoke(test).star(args='args'
-        ).constants(3, b='b').specs(c='c'
-        ).star(args='args2', kwargs='kwargs')
+    spec = (Invoke(test).star(args='args')
+            .constants(3, b='b').specs(c='c')
+            .star(args='args2', kwargs='kwargs'))
```

Comment:
this is better than the one before, but i think one per line might read nice here.